`@versioned` is masking mypy errors in *_laplace calls

See https://github.com/opendp/opendp/issues/1263#issuecomment-1948973263

When the `@versioned` decorator is removed, we get a few new mypy errors on calls to `make_base_discrete_laplace` and `make_base_laplace`

#1263 is blocked until this is fixed. Either make a branch from that branch, or temporarily remove the generated annotation just to see the mypy errors locally.

- [ ] src/opendp/context.py:684: error: Missing positional arguments "input_metric", "scale" in call to "make_base_laplace"  [call-arg]
- [ ] src/opendp/context.py:684: error: Argument 1 to "make_base_laplace" has incompatible type "float"; expected "Domain"  [arg-type]
- [ ] test/test_comb.py:58: error: Unexpected keyword argument "D" for "make_base_discrete_laplace"  [call-arg]
- [ ] src/opendp/measurements.py:230: note: "make_base_discrete_laplace" defined here

Improving the test coverage here would also be good.

--- a/python/src/opendp/context.py
+++ b/python/src/opendp/context.py
@@ -681,7 +681,7 @@ def _translate_measure_distance(d_from, from_measure, to_measure):
             d_out=d_from,
             T=float,
         )
-        return make_base_laplace(scale).map(constant)
+        return make_base_laplace(*space, scale).map(constant)
 
     if from_to == (
         "FixedSmoothedMaxDivergence",

--- a/python/test/test_comb.py
+++ b/python/test/test_comb.py
@@ -55,7 +55,9 @@ def test_make_basic_composition_leak():
 
     # choose a vector-valued mechanism that should run quickly for large inputs
     # we want to add as little noise as possible, so that execution time is small
-    meas = dp.m.make_base_discrete_laplace(scale=1e-6, D=dp.VectorDomain[dp.AtomDomain[int]])
+    domain = dp.VectorDomain[dp.AtomDomain[int]]
+    metric = ... # TODO!
+    meas = dp.m.make_base_discrete_laplace(domain, metric, 1e-6)
 
     # memory usage remains the same when this line is commented,
     # supporting that AnyObject's free recursively frees children

--- a/python/test/test_comb.py
+++ b/python/test/test_comb.py
@@ -55,9 +55,8 @@ def test_make_basic_composition_leak():
 
     # choose a vector-valued mechanism that should run quickly for large inputs
     # we want to add as little noise as possible, so that execution time is small
-    domain = dp.VectorDomain[dp.AtomDomain[int]]
-    metric = ... # TODO!
-    meas = dp.m.make_base_discrete_laplace(domain, metric, 1e-6)
+    space = dp.vector_domain(dp.atom_domain(T=int)), dp.l1_distance(T=int)
+    meas = space >> dp.m.then_laplace(0.)
 
     # memory usage remains the same when this line is commented,
     # supporting that AnyObject's free recursively frees children
@@ -67,8 +66,8 @@ def test_make_basic_composition_leak():
     # memory would jump by ~40mb every iteration
     for i in range(1000):
         print('iteration', i)
-        meas([0] * 10_000_000)
-    
+        meas([0] * 1_000_000)
+
 
 def test_make_basic_composition_approx():
     input_space = dp.atom_domain(T=float), dp.absolute_distance(T=float)

